examples/web/index.html

```diff
@@ -220,11 +257,31 @@ <h2 class="typography-heading-medium-text mt-1100 mb-800">TextField</h2>
                 <input type="text" id="textfieldInlineFirst" class="TextField__input" placeholder="Placeholder" value="Filled" />
               </div>
               <div class="TextField">
-                <label for="textfieldInlineSecond" class="TextField__label TextField__label--hidden">Hidden Label</label>
-                <input type="text" id="textfieldInlineSecond" class="TextField__input" placeholder="Placeholder" value="Filled" />
+                <label for="textfieldInlinePassword" class="TextField__label TextField__label--hidden">Hidden Label</label>
+                <div class="TextField__password-toggle">
+                  <input type="password" id="textfieldInlinePassword" class="TextField__input" value="Password" />
+                  <button type="button" class="TextField__password-toggle__button" role="switch" aria-pressed="false" aria-label="Show password" data-toggle="password">
+                    <span class="TextField__password-toggle__icon" aria-hidden="true"></span>
+                  </button>
+                </div>
               </div>
               <button type="button" class="Button Button--primary">Button</button>
             </div>
+            <script type="text/javascript">
+              const handlePasswordToggleClick = (event) => {
```

Comment:
⚠️ This needs to be refactored once #250 is merged.

Reply:
Then I should merge this first.